toolkit/tools/imagecustomizer/docs/cli.md

```diff
@@ -0,0 +1,83 @@
+# Mariner Image Customizer command line
+
+## --help
+
+Displays the tool's quick help.
+
+## --build-dir=DIRECTORY-PATH
+
+Required.
+
+The directory where the tool will place its temporary files.
+
+## --image-file=FILE-PATH
+
+Required.
+
+The base image file to customize.
+
+Supported image file formats: vhd, vhdx, qcow2, and raw.
+
+## --output-image-file=FILE-PATH
+
+Required.
+
+The image format of the output customized image file.
+
+Options: vhd, vhdx, qcow2, and raw.
+
+## --output-image-format=FORMAT
```

Comment:
Left this comment elsewhere, sorry have not yet caught up. But, do we really need the output-image-format when similar information is passed through output-image-file?

Reply:
I think the problem is that some of the image formats don't have a standard filename extension. Raw images are the prime example of this.